src/njoy21/input/DTFR/Card4/HollerithNames/test/HollerithNames.test.cpp

```diff
@@ -0,0 +1,58 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "HollerithNames input values",
+         "[Card4], [HollerithNames]" ){
+  Argument< DTFR::Card3::Iptotl > iptotl;
+  iptotl.value = 6 ;
+
+ GIVEN( "the correct number of names" ){
+
+    WHEN( "the names are of correct length" ){
+      iRecordStream< char> iss( std::istringstream("pheat1 edits dtfr") );
```

Comment:
They don't need the single quotes unless there's embedded whitespace, e.g.
```
no_need_for_quotes
```
```
'this needs quotes'
'they can even
be split over multiple lines'
```
The quotes are stripped off while parsing, but that's tested elsewhere